config.go

```diff
@@ -372,6 +374,18 @@ func loadConfig() (*config, error) {
 		fmt.Fprintln(os.Stderr, err)
 		return nil, err
 	}
+	if cfg.Autopilot.ConfTarget < 0 {
+		str := "%s: autopilot.conftarget must be non-negative"
```

Comment:
I realize that this pattern was established before your change, but could you introduce a helper function to eliminate the redundancy of these four "if" blocks? You'd end up with the much more compact and readable
```go
if cfg.Autopilot.ConfTarget < 0 {
    return nil, fmtAndPrintError("%s: autopilot.conftarget must be non-negative", err)
}
```
repeated four times instead.